### DISCUSSION TITLE
Wallect connect V2 switch network issue

### DISCUSSION THREAD

**Discussion by navdeep1998** [WalletConnect/walletconnect-monorepo#3148] [Developer Support]
Hi
After connection with wallet connect I'm unable to switch  on correct network that i've declare in wagmi config.I anybody have integrated switch network hook .

**Comment by SaadNiaziDev** [comment]
There must be error on console i guess. If you are using useSwitchNetwork hook from wagmi, it requires that chain to be already added to walletapp.
What are you using? Like wagmi version, switch network on which App? Provide details maybe i can help you
